lib/message/utils.js

```diff
@@ -39,6 +39,10 @@ export function createMessage(source) {
 }
 
 export function signMessage(options) {
+    if (typeof options.data === 'string') {
+        options.message = createMessage(options.data);
```

Comment:
You probably want `getCleartextMessage` here, since that was the default for signing strings. Also I think you'll need to handle `Uint8Array`s in both `encryptMessage` and `signMessage` as well (you can use `createMessage` for both).

Reply:
@twiss changes applied in this commit: 23fcc8b